cog/ctf.py

```
@@ -211,8 +206,14 @@ async def callback(self, interaction: discord.Interaction) -> None:
                                     ephemeral=True, embeds=[embed]
                                 )
                                 return
+
                             # else 未曾回答過，送獎勵
```

Ruff (0.8.2)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)